less/home-page.less

```
@@ -46,9 +59,19 @@
     display: flex;
     justify-content: center;
 
+    @media(max-width: @mobile) {
+      width: 100%;
+      flex-direction: column-reverse;
+    }  
```

After I found a fix for the mobile button, the padding was breaking it so I trimmed a bit off. 
```suggestion
    padding: 50px 10px;
    }  
```